tests/app/conftest.py

```diff
@@ -189,3 +193,33 @@ def current_location():
 @pytest.fixture
 def mock_autoescape_context(mocker):
     return mocker.Mock(autoescape=True)
+
+
+@pytest.fixture(name="mocked_response_content")
+def mocked_response_content_fixture(mocker):
+    decodable_content = Mock()
+    decodable_content.decode.return_value = b"{}"
+    mocker.patch("requests.models.Response.content", decodable_content)
+
+
+@pytest.fixture(name="mocked_make_request_with_timeout")
+def mocked_make_request_with_timeout_fixture(
+    mocker, mocked_response_content  # pylint: disable=unused-argument
```

Comment:
I think you can use `usefixtures` instead of passing as an arg and disabling pylint

Reply:
I can't get this to work, I think maybe because it is being used inside another fixture 😕

Reply:
🙁 yup, I can't think of any way to make this work having looked closer. I think its something to do with the way the decorators wrap the function, it doesn't seem to preserve the original signature so composition fails. How annoying 😄 
so I retract that one and I think what you've done makes the most sense

Reply:
that said, why use
```
@pytest.fixture(name=x)
def y()
```
instead of just 
```
@pytest.fixture
def x()
```
?

Reply:
Yeah fair enough not sure why this was done originally so have updated just to use @pytest.fixture 👍